compiler-rt/test/sanitizer_common/TestCases/Windows/dump_registers_aarch64.cpp

```diff
@@ -17,7 +17,7 @@ int main() {
   // CHECK-DUMP-NEXT:x16 = {{0x[0-9a-f]+}}  x17 = {{0x[0-9a-f]+}}  x18 = {{0x[0-9a-f]+}}  x19 = {{0x[0-9a-f]+}}
```

Comment:
Commenting on the context above - the test runs with `--strict-whitespace`, but I believe the whitespace might not be right.

Both in the original incarnation in #112254, and in the revised form in #112305, registers below 10 are spaced as `x0<space><space>= ...`, while the test above seems to assume the spacing to be `<space>x0<space>= ...`.

Additionally, in the original form in #112254 we did have spaces between the elements (but no line wrapping). In the revised form in #112305, we do have line wrapping as the test expects, but we seem to have lost the spaces between elements.

I appreciate the contributions, but if the tests aren't ever actually tested to work correctly, then please raise that during review, to ask someone to test it for you before merging!

Do @DavidSpickett or @omjavaid have an environment handy where you can run these tests?

Reply:
I don't think we run compiler-rt tests on production buildbots but iirc Omair was running it on his own instance?